certificate-policy.md

```diff
@@ -841,289 +811,247 @@ All persons filling trusted roles shall be selected on the basis of loyalty, tru
-
-**Note**: The FPKIPA will follow individual Application Trusted Root Program requirements to report security concerns.
+CAs shall have an Incident Response Plan and a Disaster Recovery Plan.  The CA is not required to publicly disclose the Incident Response Plan and Disaster Recovery Plan but shall make the plans available to the CA's Qualified Auditor upon request. 
+
+The FPKIPA shall be notified by the CAs operating under this policy of any incident. An incident is defined as a violation or imminent threat of violation of this CP, the CA's CPS, Memorandums of Agreements, or any other document that governs the operations of the CA. An incident may include but is not limited to the following:
+
+- CA private key compromise
+- Suspected or detected compromise of the CA including the certificate status services required of the CA Repository
+- Physical or electronic penetration of the CA including the certificate status services required of the CA Repository
+- Successful denial of service attacks on the CA including the certificate status services required of the CA Repository
+- Suspected or detected issuance of certificates used for unethical purposes such as (but not limited to) promoting malware or illegal software
+- A known or reasonably known, publicly reported compromise of the CA including the certificate status services required of the CA Repository
+- Any certificate issuance not in compliance with this CP, this CP's certificate profiles, or the CA's CPS
+
+The CA shall notify the FPKIPA within 24 hours from the time the incident was discovered.  An initial security incident report shall be submitted to the FPKIPA and shall include the following information:
+1. Which CA was affected by the incident
+1. When the incident was discovered
+1. How the incident was discovered 
+1. If available and applicable, any evidence of attribution for the incident
+1. The CA's interpretation of the incident
+1. A complete list of all certificates that were either mis-issued or not compliant with this CP and the CA's CPS as a result of the incident.
+
+A final security incident report shall be submitted at a date specified by the FPKIPA and shall include the following information:
+1. A complete timeline of events 
+1. A root cause analysis
+1. Remediation actions implemented to address the underlying root cause including specific technical or procedural changes, and any updates to the CA's CPS
+1. Proof the mis-issued certificates were revoked
+1. A statement that the incident has been fully remediated
+
+In coordination with the CA, the FPKIPA may conduct the following activities as part of an incident response:
+- Publicly publish a final redacted incident report on an approved government website
```

Comment:
I would rephrase to allow for redaction to be optional, and for the FPKI to publish additional copies on non-government websites (such as Bugzilla) or on non-web areas (such as an email to m.d.s.p). 

So for example:

> - Publicly publish a final incident report in one or more internet-accessible locations, with information redacted as necessary.